logicrunner/logicrunner_test.go

```diff
@@ -63,25 +68,44 @@ import (
 	"github.com/insolar/insolar/logicrunner/goplugin/goplugintestutils"
 	"github.com/insolar/insolar/testutils"
 	"github.com/insolar/insolar/testutils/testmessagebus"
-	"github.com/stretchr/testify/assert"
-	"github.com/ugorji/go/codec"
 )
 
-var icc = ""
-var runnerbin = ""
 var parallel = false
 
-func TestMain(m *testing.M) {
+type LogicRunnerFuncSuite struct {
+	suite.Suite
+
+	runnerBin    string
+	icc          string
+	contractsDir string
+}
+
+func FindContractsDir() (string, error) {
+	_, file, _, ok := runtime.Caller(0)
+	if !ok {
+		return "", errors.New("failed to find folder")
+	}
+
+	// we're located in /logicrunner/logicrunner_test.go file, so we must take two basenames
+	projectRoot := path.Dir(path.Dir(file))
+	contractsDir := path.Join(projectRoot, "application", "contract")
```

Comment:
why not GetRealApplicationDir ?

Reply:
`GetRealApplicationDir` scans every directory in `GOPATH` and tries to deduce what is the insolar dir.
Better way is to read current directory of current file and that'll be project path (and that we won't look into another copy of insolar repo)
But I can't modify `GetRealApplicationDir`, since it's used in code, not only in tests.